src/symbols/Edge.tsx

```diff
@@ -99,6 +99,8 @@ export interface EdgeProps {
   onPointerOut?: (edge: InternalGraphEdge) => void;
 }
 
+const MULTI_EDGE_OFFSET_FACTOR = 0.7;
```

Comment:
Could add this as a prop somewhere @amcdnl ? Not sure if it's something ppl want to customize and I didn't wanna just add extra noise to the prop list

Here's what it does 
![image](https://github.com/reaviz/reagraph/assets/40581813/bff709dc-6514-43cd-b0dd-563b75c3d4d7)

![image](https://github.com/reaviz/reagraph/assets/40581813/a358c006-3eda-4def-8ad4-f93983d66eaf)

Reply:
Ya I wouldn't expose for now